darts/models/prophet.py

```diff
@@ -93,3 +96,10 @@ def predict(self, n: int) -> TimeSeries:
 
         forecast = predictions['yhat'][-n:].values
         return self._build_forecast_series(forecast)
+
+    @property
+    def params(self) -> Dict[Any, Any]:
```

Comment:
The return type is collections.OrderedDict, and comment is wrong. (Exponential Smoothing > Prophet)
I'll fix it later...